### PR TITLE
Enhacement of Contribution file 

### PR DESCRIPTION
**Pull request by ayan-joshi**
First off, thanks for taking the time to contribute!

## Please Complete the Following

- [ ] I read [CONTRIBUTING.md](https://github.com/Cyclenerd/static_status/blob/master/CONTRIBUTING.md)
- [ ] I used tabs to indent
- [ ] I checked my code with [ShellCheck](https://www.shellcheck.net/)

## Notes

Gave it a better explanation @Cyclenerd

### CODE REVIEW

**Comment by Cyclenerd** [comment]
Please check titles and list styling.

**Comment by Cyclenerd** [comment]
No, you did not. Forget to push?

**Comment by ayan-joshi** [comment]
> No, you did not. Forget to push?

oh sorry now its been pushed

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,66 +1,71 @@
-# How to Contribute
+# Contributing to the Project
 
-First off, thanks for taking the time to contribute!👏
+Thank you for considering contributing to our project! Your help and involvement are highly appreciated.
+This guide will help you get started with the contribution process.
+
+## Table of Contents
+
+1. [Fork the Repository](#fork-the-repository)
```
Comment (Cyclenerd) [inline CONTRIBUTING.md:8]: Links are broken

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,66 +1,71 @@
...
 ## Create a Pull Request 🌟
-
-8. Go to your forked repository on GitHub and click on the "New Pull Request" button. This will open a new pull request to the original repository.
+Go to your forked repository on GitHub and click on the "New Pull Request" button.
+This will open a new pull request to the original repository.
 
 ## Coding Style 📝
+When contributing to the project, please adhere to the following coding style guidelines:
+
+-Variables must be in uppercase and should begin with MY_.
```
Comment (Cyclenerd) [inline CONTRIBUTING.md:60]: List is broken.

**Comment by Cyclenerd** [comment]
Please recheck. i left you comments to clarify what i exactly mean.

**Comment by ayan-joshi** [comment]
> Please recheck. i left you comments to clarify what i exactly mean.

Ok got it and made changes respect to it

**Comment by Cyclenerd** [comment]
Links were still broken. I have fixed it. Thank you 👍

**Pull request merged**